java/pom.xml

```diff
@@ -234,6 +246,7 @@
     </profiles>
 
     <build>
+	    <directory>${project.build.directory}</directory>
```

Comment:
nit: indentation appears to be off here.

Reply:
Why are we setting this at all?

Reply:
Under normal flow it will be the default directory but when we are in compose we want the user to be able to work on different branches and with different cuda versions. By setting this value we are able to control where the artifacts will be copied. Is there a better way to do this?

Reply:
if there's a way to conditionally add the `<directory>` tag, that will be ideal